tests/account_test.js

```diff
@@ -0,0 +1,12 @@
+Feature('Guestbook');
+    const r_ship = Math.floor(Math.random() * 5) + 1
+Scenario('verify username and add entry to guestbook',  ({ I }) => {
+    I.amOnPage('/us/obituaries/chicagotribune/name/virginia-gruchalski-obituary?pid=196167379');
+    I.see('Virginia','.Heading-sc-4h3nqe-0')
+    I.scrollIntoView('#guestbook')
+    I.fillField('textarea[name=_GuestbookForm_Message]', 'Sorry for your loss')
+    I.click(`div.Box-sc-ucqo0b-0.jjAWvY > div > div:nth-child(${r_ship}) > button`)
+    I.fillField('input[name=_GuestbookForm_From]', 'Duran')
+    I.fillField('input[name=_GuestbookForm_Email]', 'dthomas@qualityworkscg.com')
+    I.click('div.Box-sc-ucqo0b-0.cbNzut > div.Box-sc-ucqo0b-0.Flex-sc-d1l2vy-0.fsrkdX > button')
+});
```

Comment:
Just curious if your onboarding project mentions the protocol our team uses for `.js` tests - even though they are not in Cucumber format, we've been adding comments as if they were in the right places.  For example, before Line 4, you'd put in a comment like `// Given I am on a Janus obituary page`, and before line 5 you'd put in a comment like `// Then I should see the correct username in the heading`.  And actually, I think these should be 2 scenarios, the first one verifying the username and the second adding the entry to guestbook.  We can sync up and chat about this - I may not be understanding the purpose of this particular exercise.

Reply:
I'm also curious about the locators - they look a bit like you're relying on xpath a lot, instead of more readable ids or classes?

Reply:
There was no mention of using the cucumber format or the protocol used by the team for this task.

I could update to include a second scenario for the other case.

XPath wasn't used for any of the selectors here

Reply:
I still wonder if there's a simpler way of defining some of these elements - maybe not! I'm not an expert on the Janus pages by any means!

Reply:
@pegfoltz , yes, that is how I first started. The onboarding will then require you to edit that test case to have the feature-step format.

Reply:
@duran-thomas like peg has mentioned. Please update to have some comments in there. The three comment statements we use is `//Given I ...` `//When I ...` `//Then I ...` . They also should be in that order. Let me know if you have any questions.

Reply:
And adding those comments in the right places now will make it easier to convert to BDD format in a later task!

Reply:
awesome comments. Only suggestion is to add some spaces between your comment and `//`
` //Then I should be able to add an entry to the guestbook` -> ` // Then I should be able to add an entry to the guestbook`

Reply:
We have a plug-in on VSCode that will enforce things like that called ESLint - please install it if you haven't already since it will ensure correct formatting of all your code!  Really handy!

Reply:
will do